### PR TITLE
linux: automatically add CONFIG_INITRAMFS_ROOT_{U,G}ID=0

### PR DESCRIPTION
**Pull request by lrusak**
This allows updating kernel configs easier.

from the docs
```
This setting is only meaningful if the INITRAMFS_SOURCE is contains a directory.
 - depends on: CONFIG_INITRAMFS_SOURCE !=""
```

### CODE REVIEW

File: packages/linux/package.mk [hunk 1]
```diff
@@ -103,6 +103,7 @@ post_patch() {
   cp $KERNEL_CFG_FILE $PKG_BUILD/.config
   if [ ! "$BUILD_ANDROID_BOOTIMG" = "yes" ]; then
     sed -i -e "s|^CONFIG_INITRAMFS_SOURCE=.*$|CONFIG_INITRAMFS_SOURCE=\"$BUILD/image/initramfs.cpio\"|" $PKG_BUILD/.config
+    sed -i '/^CONFIG_INITRAMFS_SOURCE=*./ a CONFIG_INITRAMFS_ROOT_UID=0\nCONFIG_INITRAMFS_ROOT_GID=0' $PKG_BUILD/.config
```
Comment (MilhouseVH) [inline packages/linux/package.mk]: `sed -i -e` ?

Comment (lrusak) [reply packages/linux/package.mk]: not needed

Comment (MilhouseVH) [reply packages/linux/package.mk]: Maybe so, but it's clearly the convention (see `sed -I -e` line immediately above).

**Comment by lrusak** [comment]
done

**Pull request merged**